runtime/doc/lua.txt

```diff
@@ -2513,6 +2513,180 @@ vim.loader.reset({path})                                  *vim.loader.reset()*
       • {path}  (`string?`) path to reset
 
 
+==============================================================================
+Lua module: vim.pack                                                *vim.pack*
+
+WORK IN PROGRESS built-in plugin manager! Early testing of existing features
+is appreciated, but expect breaking changes without notice.
+
+Manages plugins only in a dedicated |packages| *vim.pack-directory*:
+`$XDG_DATA_HOME/nvim/site/pack/core/opt`. Plugin's subdirectory name matches
+plugin's name in specification. It is assumed that all plugins in the
+directory are managed exclusively by `vim.pack`.
+
+Uses Git to manage plugins and requires present `git` executable of at least
+version 2.36. Target plugins should be Git repositories with versions as named
+tags following semver convention `v<major>.<minor>.<patch>`.
+
+Example workflows ~
+
+Basic install and management:
+• Add |vim.pack.add()| call(s) to 'init.lua': >lua
+
+    vim.pack.add({
+      -- Install as "plugin1"/"plugin2" and use greatest available version
+      'https://github.com/user/plugin1',
+      { source = 'https://github.com/user/plugin2' },
+
+      -- Specify plugin's name
+      { source = 'https://github.com/user/generic-name', name = 'plugin3' },
```

Comment:
Maybe clarify here that if `name` is omitted it's inferred from the URL?

```suggestion
      -- Specify plugin's name (here the plugin will be called "plugin3"
      -- instead of "generic-name")
      { source = 'https://github.com/user/generic-name', name = 'plugin3' },
```